llvm/include/llvm/Support/PropertySetIO.h

```diff
@@ -0,0 +1,259 @@
+//==-- PropertySetIO.h -- models a sequence of property sets and their I/O -==//
```

Comment:
Is support where this should go? Feels more like an object thing.

Reply:
I think 'Support' is a better option. It is just yet another data format.

Thanks